pkg/collector/corechecks/snmp/common/utils.go

```diff
@@ -42,3 +43,36 @@ func CopyStrings(tags []string) []string {
 func GetAgentVersionTag() string {
 	return "agent_version:" + version.AgentVersion
 }
+
+// NormalizeNamespace applies policy according to hostname rule
+func NormalizeNamespace(namespace string) (string, error) {
+	var buf bytes.Buffer
+
+	// namespace longer than 100 characters are illegal
+	if len(namespace) > 100 {
+		return "", fmt.Errorf("namespace is too long, should contain less than 253 characters")
```

Comment:
No strong opinion if it should be 100 or 253 but we should have the error message to match the condition